project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -54,9 +54,24 @@
 
   /**
    * @return the coordinates of the physical place.
-  */
+   */
   public abstract LatLng location();
 
+  /**
+   * @return the URL of the place’s google page.
+   */
+  public abstract String googleUrl();
+
+  /**
+    * @return the place's placeId as defiened on Google Places.
+    */
+  public abstract String placeId();
+
+  /**
+    * @return the place's business status
```

Comment:
Please document here as well the possible values for businessStatus.

Reply:
Added